test/util.py

```diff
@@ -217,7 +256,8 @@ def create_sample_video_as_blob(num_frames=NUM_FRAMES):
 
 def copy_sample_videos_to_upload_dir():
     shutil.copyfile(
-        "data/ua_detrac/ua_detrac.mp4", os.path.join(UPLOAD_DIR, "ua_detrac.mp4")
+        "data/ua_detrac/ua_detrac.mp4",
```

Comment:
Will these hard-coded paths work on Windows?

Reply:
They might not. I’ll fix these. Looks like we have a lot of instances in the this file. Can be done in a separate PR.